code/script.js

```diff
@@ -0,0 +1,89 @@
+const projects = document.getElementById("projects");
+const userProfile = document.getElementById("userProfile");
+const chart = document.getElementById("chart");
+const user_URL = "https://api.github.com/users/Sherin-Susan-Thomas";
+const repo_URL = "https://api.github.com/users/Sherin-Susan-Thomas/repos";
+
+const options = {
+  method: "GET",
+  headers: {
+    Authorization: `token ${API_TOKEN}`,
+  },
+};
+
+//To fetch profile data
+const user = () => {
+  fetch(user_URL, options)
+    .then((res) => res.json())
+    .then((data) => {
+      console.log("data", data);
+      userProfile.innerHTML += `
+      <h1> GitHub Tracker <br> </h1> 
+      <img class= "user" src = ${data.avatar_url}>
+      <h2 class = "user-name" id= "userName"> ${data.name}  </h2> 
+      <h4> ${data.bio} </h4>  <br>
+      <h4>  Follow <a href="${data.html_url}">   <i class="fa fa-github" aria-hidden="true"></i> </a>   </h4> 
+     
+     `;
+    });
+};
+user();
+
+// To fetch repositories
+const userRepo = () => {
+  fetch(repo_URL, options)
+    .then((res) => res.json())
+    .then((userData) => {
+      console.log("userData", userData);
+      const filteredRepos = userData.filter(
+        (item) => item.fork == true && item.name.includes("project-") // to filter technigo projects
```

Comment:
item.name.includes might be true if a project includes this string in its name. A better fit here might be item.name.startsWith